notes/stories/film/film-scifi-1980s.st.txt

```diff
@@ -668,7 +668,7 @@ coping with getting dumped [Laura did not take Pete dumping her like a sack of p
 father and son [A father listened to his teenage son and daughter explain that there was a sea serpent in the lake.]
 father and daughter [A father listened to his teenage son and daughter explain that there was a sea serpent in the lake.]
 brother and sister [A teenage brother and sister pair went out in the middle of the night to photograph the sea serpent.]
-superstition [Some dismissed villager reports of evil spirits in the lake as local superstition.]
+modern superstition [Some dismissed villager reports of evil spirits in the lake as local superstition.]
```

Comment:
spirits in a lake sound more like folklore...

Reply:
Changed to *folklore* with gusto.